stable/spring-cloud-data-flow/README.md

```diff
@@ -171,4 +185,31 @@ The following tables list the configurable parameters and their default values.
 | ---------------------------- | --------------------------------------- | ------------------------- |
 | features.streaming.enabled   | Enables or disables streams             | true
 | features.batch.enabled       | Enables or disables tasks and schedules | true
-
+| features.monitoring.enabled  | Enables or disables monitoring          | false
+
+### Grafana
+
+| Parameter                            | Description                                                  | Default                    |
+| ------------------------------------ | ------------------------------------------------------------ | -------------------------- |
+| grafana.service.type                 | Service type to use                                          | LoadBalancer
+| grafana.admin.existingSecret         | Existing Secret to use for login credentials                 | scdf-grafana-secret
+| grafana.admin.userKey                | Secret userKey field                                         | admin-user
+| grafana.admin.passwordKey            | Secret passwordKey field                                     | admin-password
+| grafana.admin.defaultUsername        | The default base64 encoded login username used in the secret | admin
+| grafana.admin.defaultPassword        | The default base64 encoded login password used in the secret | password
+| grafana.extraConfigmapMounts         | ConfigMap mount for datasources                              | scdf-grafana-ds-cm
+| grafana.dashboardProviders           | Dashboard provider for imported dashboards                   | default
+| grafana.dashboards                   | Dashboards to auto import                                    | SCDF Apps, Streams & Tasks
+
+### Prometheus
+| Parameter                                    | Description                                        | Default                                |
+| -------------------------------------------- | -------------------------------------------------- | -------------------------------------- |
+| prometheus.server.global.scrape_interval     | Scrape interval                                    | 10s
+| prometheus.server.global.scrape_timeout      | Scrape timeout                                     | 9s
+| prometheus.server.global.evaluation_interval | Evaluation interval                                | 10s
+| prometheus.extraScrapeConfigs                | Additional scrape configs for proxied applications | `proxied-applications` & `proxies` jobs
+| prometheus.podSecurityPolicy                 | Enable or disable PodSecurityContext               | true
+| prometheus.alertmanager                      | Enable or disable alert manager                    | false
```

Comment:
Any particular release why they are not `alertManager` or `pushGateway`? Just a nitpick from consistency pov.

Reply:
those are prom properties, not ours